### DISCUSSION TITLE
Embed Local Vector Data Upload in Exported HTML

### DISCUSSION THREAD

**Discussion by ar-siddiqui** [opengeos/leafmap#103] [Q&A] [answered]
I have a map that displays the toolbox to load local vector data in Jupiter. When I export it to HTML that toolbox goes away. I need to create a map and export it as HTML with the option for people to load their files and visualize on top of my map layers. Is this possible through leafmap?

**Comment by giswqs** [comment, marked as answer]
Unfortunately, the exported HTML won't support the upload functionality. You will need voila to deploy the app. Check out tutorials 26, 27, 28, 42 at https://geemap.org/tutorials. Those tutorials are for the geemap package, and they are applicable to leafmap as well.
